changing image for Community call

clcoffey opened the pull request:
The image for the latest community call is the same as the one for the last community call. I am updating it to the correct image.

tsadler1988 reviewed (approved):
Ran locally and all looks good!

lenucksi reviewed (approved):
Looks fine, thanks!

rrrutledge commented:
Thank you!

clcoffey committed:
changing image for Community call

The image for the latest community call is the same as the one for the last community call. I am updating it to the correct image.

claredillon reviewed (approved):
thanks!
